packages/jaeger-ui/src/selectors/trace.test.js

```diff
@@ -13,7 +13,7 @@
 // limitations under the License.
 
 import { followsFromRef } from './trace.fixture';
-import { getSpanId } from './span';
+import getSpanId from './span';
```

Comment:
if this is the only place, let's just replace it with span.ID and delete the selector

Reply:
Sounds good!